<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dinForm</class>
 <widget class="QWidget" name="dinForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>135</y>
     <width>116</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Display 
Wakeup</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_on_off">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>135</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Emergency</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nzl1_locking">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>90</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>225</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Smoke</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_leakage">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>180</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_smoke">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>225</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>225</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SPD Alarm</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_spd_alarm">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>225</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>135</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Insulation</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_emergency">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NOZZLE-1
Lifted</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Leakage 
Current</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nz1_lifted">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nozzle 
Locking</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_insulation">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>135</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="dibackButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>751</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Digital input</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startDinButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>330</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Start_Reading</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopDinButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>330</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop_Reading</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readDinNowButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Read_Now</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_vld">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>270</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>270</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>VLD out</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>EV
Connection</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_evConnection">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
